src/caffe/layers/cudnn_deconv_layer.cpp

```diff
@@ -214,15 +218,18 @@ void CuDNNDeconvolutionLayer<Dtype>::Reshape(
         break;
       }
     }
-    if(!found_conv_algorithm) LOG(ERROR) << "cuDNN did not return a suitable algorithm for deconvolution.";
-    else{
-      // choose backward algorithm for filter
-      // for better or worse, just a fixed constant due to the missing
-      // cudnnGetConvolutionBackwardFilterAlgorithm in cuDNN version 8.0
-      bwd_filter_algo_[i] = CUDNN_CONVOLUTION_BWD_FILTER_ALGO_0;
-      //twice the amount of the forward search to be save
-      workspace_bwd_filter_sizes_[i] = 2 * workspace_fwd_sizes_[i];
-    }
+
+    if(!found_conv_algorithm) CUDNN_CHECK(cudnnStatus_t::CUDNN_STATUS_BAD_PARAM);
```

Comment:
Coding style! Body of `if` must be next line
```c++
if(!found_conv_algorithm)
  CUDNN_CHECK(cudnnStatus_t::CUDNN_STATUS_BAD_PARAM);
```

Reply:
fml, I won't make a new release for a whitespace fix.

also this is a caffe fork, where is the coding style doc?